EvolvSwiftSDK/EvolvSwiftSDKTests/TestingData/genome.json

```diff
@@ -0,0 +1,80 @@
+{
+    "web": {
```

Comment:
I would prefer if the keys were renamed so this didn't give the impression the values used for a web experiment were relevant here -- the only extra complexity is that the key is returning an object

Reply:
You mean to change key with name "web" to something different when parsing JSON and creating Model with a new name field (property) accordingly?

Reply:
No, I just mean changing your example genome to use some names that are more general and not tuned for web experiments. You grabbed this genome to get a more complex genome to read in and test. Really all we're looking for is the ability to have the variant value be an object isn't it?

Reply:
Ok, understood